Kitodo/src/test/java/org/kitodo/production/services/workflow/WorkflowControllerServiceIT.java

```diff
@@ -300,7 +316,7 @@ public void shouldCloseAndAssignNextForProcessWithParallelTasks() throws Excepti
 
     @Test
     public void shouldCloseForProcessWithScriptParallelTasks() throws Exception {
-        assumeTrue(!SystemUtils.IS_OS_WINDOWS && !SystemUtils.IS_OS_MAC);
+        assumeTrue(!SystemUtils.IS_OS_WINDOWS);
```

Comment:
Is Mac no longer required? Or has the behavior changed, and are you testing with Mac?

Reply:
I am developing and testing on MacOS, and this test does indeed work on the system, so there is no need skip it on Mac, IMO.